### ISSUE TITLE
v0.3.7

### ISSUE THREAD

**Issue by cirobr** [cirobr/TinyMachines.jl#23]
@JuliaRegistrator register

Release notes:

## Breaking changes

- Remove final activation from all models (expose logits).
